python/lsst/afw/table/schema/schemaContinued.py

```diff
@@ -0,0 +1,204 @@
+#
+# LSST Data Management System
+# Copyright 2017 LSST/AURA.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+from __future__ import absolute_import, division, print_function
+
+__all__ = ["Key", "Field", "SchemaItem"]
+
+from past.builtins import basestring
+from builtins import str as futurestr
+from future.utils import with_metaclass
+
+import numpy as np
+import fnmatch
+import re
+import collections
+import astropy.units
+
+from ...geom import Angle
+from lsst.utils import continueClass, TemplateMeta
+
+from .schema import _Key, _Field, _SchemaItem, Schema
+
+# Objects we prefer to use over the C++ string name for
+# Key/Field/SchemaItem types.
+_dtypes = {
+    "String": str,
+    "U": np.uint16,
+    "I": np.int32,
+    "L": np.int64,
+    "F": np.float32,
+    "D": np.float64,
+    "Angle": Angle
+}
+
+
+class Key(with_metaclass(TemplateMeta, object)):
+    pass
+
+
+class Field(with_metaclass(TemplateMeta, object)):
+    pass
+
+
+class SchemaItem(with_metaclass(TemplateMeta, object)):
+    pass
+
+
+def _registerInstantiations(abc, types):
+    """Iterate over a private dict (filled by template instantiations in C++)
+    to register template instantiations a TemplateMeta ABCs.
+
+    If an entry for the type string exists in _dtypes, we use that instead of
+    the string as the key, and use the string as an alias.
+    """
+    for k, v in types.items():
+        dtype = _dtypes.get(k, None)
+        if dtype is not None:
+            abc.register(dtype, v)
+            abc.alias(k, v)
+        else:
+            abc.register(k, v)
+
+
```

Comment:
Please add comment here explaining where `_Key`, `_Field` and `_SchemaItem` come from and what they contain.